runcrud.bat

```diff
@@ -0,0 +1,33 @@
+call gradle build
```

Comment:
W tego rodzaju skryptach powinieneś korzystać z gradle wrappera (`gradlew`). Jeśli używasz polecenia `gradle`, to wymagasz, aby użytkownik Twojego skryptu miał zainstalowany na swoim komputerze Gradle. Po to jest gradle wrapper (`gradlew`), żeby nie musiał.